addons/modules/stringtable.xml

```diff
@@ -1278,6 +1278,18 @@
             <Polish>Odgarnizonuj grupę</Polish>
             <Japanese>非歩哨グループ</Japanese>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleUnpackStaticWeapon">
+            <English>Unpack Static Weapon</English>
+        </Key>
+        <Key ID="STR_ZEN_Modules_ModuleUnpackStaticWeapon_Unit">
+            <English>Unit must carry a weapon bag</English>
+        </Key>
+        <Key ID="STR_ZEN_Modules_ModuleUnpackStaticWeapon_Group">
+            <English>Unit group must contain two compatible weapon bags</English>
+        </Key>
+        <Key ID="STR_ZEN_Modules_ModuleUnpackStaticWeapon_Facing">
+            <English>Static Weapon Facing</English>
```

Comment:
Not sure on this: maybe "Weapon Direction" or "Static Weapon Direction" would be better.

Reply:
Certainly! It was a carryover from Company of Heroes "set facing".